scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -100,9 +104,10 @@ object ScanCode : LocalScanner() {
         }
 
         val result = getResult(resultsFile)
+        val hasOnlyMemoryErrors = mapMemoryErrors(result)
 
         with(process) {
-            if (exitValue() == 0 || hasOnlyTimeoutErrors(result)) {
+            if (exitValue() == 0 || hasOnlyMemoryErrors || hasOnlyTimeoutErrors(result)) {
```

Comment:
Shouldn't we treat them equally here like `hasOnlyMemoryOrTimeoutErrors`?

Reply:
Basically you're right, but in practice I'v never seen both memory and timeout errors in the same scan.